test/services/postcard.spec.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+var Config  = require('../../config');
```

Comment:
We usually keep our local `require`s in a separate block from our global `require`s, so you should just add a new line before this line.